contrib/cirrus/logcollector.sh

```diff
@@ -60,6 +60,7 @@ case $1 in
                     container-selinux
                     libseccomp
                     netavark
+                    passt
```

Comment:
I'll show my pasta ignorance, is this correct, or did you accidentally type the German word for "fits" here?

Reply:
no there are two binaries, passt and pasta (see https://passt.top/passt/about/#motivation)
We need pasta but they are both part of the same rpm called passt.

Reply:
@Luap99 thanks for the info. I looks all over in Google last night and kept running into discussions on the meaning of "passt" in German versus any code-related stuff.

Reply:
@TomSweeneyRedHat I just came across this while looking for another merge request, so I guess I owe you the fully story ;) I'm not a native German speaker, but I'll try to be accurate.

So, yes, "passt" means "fits", being the third person singular of the verb "passen", "to fit" (cf. passable, also via French).

Especially in southern Germany, though, it's often used to stress that something just happens to barely fit or work, being rather far from a perfect fit or execution, sometimes humorously, alone or in sentences such as "sitzt, passt, wackelt und hat Luft": "sits, fits, wobbles and has air [around it]".

This pejorative sense is embodied by the current [logo](https://passt.top/passt.webp): "passt" is something one could victoriously declare after taping drain pipes together. I think it's rather solid now, but I started it as a hack, and that was "passt" for me. If you go all the way to the first git commit of that project, it gets even more explicit.

I hope this helps. :)